portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,6 +15,8 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.*;
```

Comment:
If this is for ArrayList, you can do: import java.util.ArrayList;
We don't usually use * in import statements, as it can be a lot of stuff you're importing at once.

Reply:
Bad habit of mine. Fixed!